libtransmission/utils.cc

```diff
@@ -306,7 +306,7 @@ uint8_t* tr_loadFile(char const* path, size_t* size, tr_error** error)
     }
 
     /* file size should be able to fit into size_t */
-    if (sizeof(info.size) > sizeof(*size))
+    if constexpr (sizeof(info.size) > sizeof(*size))
```

Comment:
if it's compile time check, maybe use `static_assert`?

Reply:
Yep that was my first though too, but  it doesn't work here: It's a two-part check, and only the first part is compile-time. :man_shrugging: